lib/components/form/mode-button.js

```diff
@@ -1,6 +1,6 @@
-import React, {PropTypes, Component} from 'react'
+import React, {PropTypes, Component, PureComponent} from 'react'
```

Comment:
Any reason PlusTransit is a PureComponent? This feels like a bit of an arbitrary decision.

Reply:
PureComponent increases performance by having a default `shouldComponentUpdate` method defined. This does a shallow comparison of the props sent to the component and will cut back on the number of renders the component does.